versioned_docs/version-2.0/concepts/admin/admin.mdx

```diff
@@ -0,0 +1,45 @@
+# Admin
+
+This conceptual guide covers topics related to managing users, organizations, and workspaces within LangSmith.
+
+## Organizations
+
+An organization is a logical grouping of users within LangSmith. Typically, there is one organization per company. An organization can have multiple workspaces.
+
+## Workspaces
+
+A workspace is a logical grouping of users within an organization that share access to the same set of resources, including tracing projects, datasets, annotation queues, and prompts.
+
+## Users
```

Comment:
We may want to add a section about authentication modes, or I can add that under the self-hosted docs specifically with the basic auth launch

Reply:
This should go under reference